pkgs/tools/misc/zf/default.nix

```
@@ -0,0 +1,51 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  zig,
+  testers,
+  zf,
+  ...
```

```suggestion
  zf
```
Arguments should be explicit.

---

done